joerd/source/ned_base.py

```diff
@@ -19,122 +20,107 @@
 from osgeo import gdal
 import urllib2
 import shutil
+import yaml
+import time
 
 
-def __download_ned_file(img_name, zip_name, base_dir, ftp_server, base_path,
-                        options):
-    logger = logging.getLogger('ned')
-    output_file = os.path.join(base_dir, img_name)
+GLOBAL_CACHE = {}
```

Comment:
Does this need to be at global scope? Can it sit inside the instance? Or are there multiple instances that share this cache?